CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.0.0](https://github.com/open-feature/go-sdk/compare/v1.1.0...v2.0.0) (2023-02-02)
+
+
+### ⚠ BREAKING CHANGES
+
+* upgrade Go to 1.18 ([#140](https://github.com/open-feature/go-sdk/issues/140))
```

Comment:
Do we still want to highlight this as breaking if we are deciding semantically that it's not?

Reply:
Perhaps not if we are introducing the compatibility table?

Reply:
OTel did it like this.
https://github.com/open-telemetry/opentelemetry-go/blob/main/CHANGELOG.md#%EF%B8%8F-notice-%EF%B8%8F-1